outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "dn" {
-  value       = nxos_vrf.l3Inst.id
+  value       = "sys/inst-${var.name}"
```

Comment:
Why did you change this to `sys/inst-${var.name}`? The idea here is to use the computed value of `nxos_vrf.l3Inst.id, which ensures that if it is used as an input for another resource/module it create an implicit dependency, which guarantees that the respective object (`l3Inst` in this case) has already been created. If we would use `sys/inst-${var.name}` instead, this would not result in the desired behavior, as the the output value would be available immediately.

Reply:
The reason I did this is because we no longer create the L3inst for the the vrf default. 
```
resource "nxos_vrf" "l3Inst" {
  count       = var.name == "default" ? 0 : 1 # l3Inst is always present for default VRF, can not be deleted
```
So for the vrf default we will fail to produce the output ```nxos_vrf.l3Inst.id``` because nxos_vrf.l3Inst does not exist.

Reply:
We can do it this way:
```
  value       = var.name == "default" ? "sys/inst-${var.name}" : nxos_vrf.l3Inst.id
```

```sys/inst-default``` always exist in the DME, so it should be ok. What to you think?